spec/models/library_entry_spec.rb

```diff
@@ -134,6 +135,49 @@
     end
   end
 
+  describe 'timestamp validation' do
+    let!(:anime) { create(:anime) }
+    let!(:library_entry) do
+      create(:library_entry, media: Anime.last, status: :current, progress: 1)
```

Comment:
with `let(:anime)` at the top of the `RSpec.describe` block, using `media: :anime` throws `NoMethodError: undefined method `primary_key' for Symbol:Class`